### ISSUE TITLE
Error token API MapTile

### ISSUE THREAD

**Issue by H4rdC0d3** [geometalab/Vector-Tiles-Reader-QGIS-Plugin#287]
When I try to connect to ***MapTiler Cloud (default entry with credits)*** an error occurs.

>Creating reader failed: Loading error: Error transferring https://api.maptiler.com/tiles/v3/tiles.json?key=6irhAXGgsi8TrIDL0211 - server replied: Invalid key
>
>URL incorrect? (HTTP Status 403)

![imagen](https://user-images.githubusercontent.com/35000021/101878860-9ec56c80-3b90-11eb-90b1-ad06b89e1fde.png)

**Comment by mnboos** [comment]
That's quite unfortunate, the key isn't valid anymore. Maybe @sfkeller or @klokan can help?

**Comment by wilhelmberg** [comment]
HTTP 403 means not allowed. @H4rdC0d3 you need to get a proper key:

![image](https://user-images.githubusercontent.com/4549888/101880219-d59c8200-3b92-11eb-8c46-89b91e52cbd2.png)

![image](https://user-images.githubusercontent.com/4549888/101880325-011f6c80-3b93-11eb-84a8-4aec8671269d.png)

**Comment by mnboos** [comment]
> 
> 
> HTTP 403 means not allowed. @H4rdC0d3 you need to get a proper key:
> 

You're right of course, but we're talking about the predefined connection here and it shouldn't happen there.

**Comment by klokan** [comment]
I am not aware of any change in our side. Definitely not intentional.
